README.md

```diff
@@ -1,32 +1,88 @@
-# Trash-app: Back-End Nodejs server 
+# Trash App
 
-This is a back-end nodejs server for Trash-app project. This server provides REST api accessing to backend data stored in both MongoDB and external API servers. 
+Trash App is a MERN prototype single page web app built to provide an alternative way to search for recycling centers that can accept specific materials. Based on kickstartcoding's [react-MERN-prototyping-starter](https://github.com/kickstartcoding/react-mern-prototyping-starter).
 
+**DO NOT** use this project in production. It exposes an API to MongoDB without
+any sort of security. The only purpose of the API is to allow rapid front-end
+prototyping without much (or any) backend modifications.
```

Comment:
This is not true. I put validations. It would never work like what originally made.  A lot secure than the default :) We do not need this paragraph.